cli/command_server_tls.go

```diff
@@ -88,7 +87,7 @@ func (c *commandServerStart) maybeGenerateTLS(ctx context.Context) error {
 	}
 
 	fingerprint := sha256.Sum256(cert.Raw)
-	fmt.Fprintf(c.out.stderr(), "SERVER CERT SHA256: %v\n", hex.EncodeToString(fingerprint[:])) //nolint:errcheck
```

Comment:
this precise format is actually important the the functioning of KopiaUI which will parse it at startup, we should not change it without verifying that KopiaUI will still work ok. If you'd like to get rid of these messages completely, I suggest adding a flag to do so but by default have them printed out.

Reply:
Thank you for spotting it. Are you referring to what is happening in `server.js`?

https://github.com/kopia/kopia/blob/a8fc80f47c9875076b1eaaa9ba0a2248a6b5c98d/app/public/server.js#L169-L172

In that case, we could avoid calling `fmt.Printf`only when the `json-log-console` is used, without adding a new flag.
I see that that option is not used by KopiaUI.

https://github.com/kopia/kopia/blob/a8fc80f47c9875076b1eaaa9ba0a2248a6b5c98d/app/public/server.js#L37-L51

This would make sense, because those `fmt.Printf` are disrupting the JSON format of the Kopia server output.
What do you think?